src/metabase/embed/settings.clj

```diff
@@ -35,7 +43,7 @@
                                          {:event                      (if new-value
                                                                         :embedding-enabled
                                                                         :embedding-disabled)
-                                          :embedding-app-origin-set   (boolean (embedding-app-origin))
+                                          :embedding-app-origin-set   (boolean (embedding-app-origins-sdk))
```

Comment:
I wouldn't change anything here. Since it would be replaced by these 2 new settings anyway:
- enable-embedding-static
- enable-embedding-sdk
- enable-embedding-interactive

I'm still not sure about these snowplow events for each of the new setting toggle, maybe @albertoperdomo could spec it whether they should stay roughly the same as `enable-embedding` or have different shapes entirely since we now can control each embed type individually, so we might want to embed different embed specific attributes.

And also originally `embedding-app-origin-set` was for interactive embedding's allowed origins which is `embedding-app-origin` (would be replaced with `embedding-app-origins-interactive` eventually), changing it here to SDK's origin doesn't make sense.